src/Services/Manager.vala

```diff
@@ -69,6 +69,7 @@ public class Bluetooth.Services.ObjectManager : Object {
                 object_manager_proxy_get_type,
                 null
             );
+            if (object_manager == null) return;
```

Comment:
Contravenes elementary code style: Should be 
```
if (object_manager == null) {
    return;
}
```

Reply:
done!